tests/unit/bokeh/server/test_contexts.py

```diff
@@ -140,14 +134,32 @@ class FakeRequest(object):
     async def test_create_session_if_needed_logout_url(self) -> None:
         app = Application()
         c = bsc.ApplicationContext(app, io_loop="ioloop", logout_url="/logout")
-        class FakeRequest(object):
-            arguments = dict(foo=10)
-        req = FakeRequest()
-        s = await c.create_session_if_needed("foo", request=req)
+        s = await c.create_session_if_needed("foo")
         session = c.get_session("foo")
         assert session == s
         assert c._session_contexts[session.id].logout_url == "/logout"
 
+    @pytest.mark.asyncio
```

Comment:
will need to rebase on master and remove the decorator now after https://github.com/bokeh/bokeh/pull/9662